### ISSUE TITLE
Update link in github config

### ISSUE THREAD

**Issue by madiedgar** [legesher/legesher#77]
**Update Github Config Link**
The link in the `.github/config.yml` is incomplete. Please update the link to include the `.html` extension at the end of the url. 

It should look like this:

``` yml
We'd love to celebrate you and your contribution to the community. Let us say "Thank You" again, by sending you a [special gift](https://legesher.io/dreamteam.html) only given to special individuals like you who have chosen to give back.
```

**Comment by npang4** [comment]
May i claim?

**Comment by madiedgar** [comment]
Absolutely! That would be great @npang4 👏 I'll assign you to it!

**Comment by madiedgar** [comment]
This was resolved in #78, closing this issue now. Thanks @npang4
